### DISCUSSION TITLE
Excessive Span of Questions Regarding Mexican Airlines - Request for Moderation

### DISCUSSION THREAD

**Discussion by Tgentil** [community/community#124339] [Other Feature Feedback, Questions, & Ideas] [answered]
Hello, community! I've noticed an unusual number of repetitive questions about the number of Mexican airlines appearing frequently in our discussions. This seems to be a form of spam and is cluttering our forum with redundant content. Can a moderator please look into this issue and possibly clean up these posts? Here are some of the links to these discussions for reference:

- [Discussion 124356](https://github.com/orgs/community/discussions/124356)
- [Discussion 124350](https://github.com/orgs/community/discussions/124350)
- [Discussion 124338](https://github.com/orgs/community/discussions/124338)
- [Discussion 124336](https://github.com/orgs/community/discussions/124336)
- [Discussion 124331](https://github.com/orgs/community/discussions/124331)
- [Discussion 124324](https://github.com/orgs/community/discussions/124324)
- [Discussion 124322](https://github.com/orgs/community/discussions/124322)
- [Discussion 124320](https://github.com/orgs/community/discussions/124320)
- [Discussion 124316](https://github.com/orgs/community/discussions/124316)
- [Discussion 124311](https://github.com/orgs/community/discussions/124311)
- [Discussion 124312](https://github.com/orgs/community/discussions/124312)
- [Discussion 124308](https://github.com/orgs/community/discussions/124308)
- [Discussion 124307](https://github.com/orgs/community/discussions/124307)
- [Discussion 124301](https://github.com/orgs/community/discussions/124301)
- [Discussion 124300](https://github.com/orgs/community/discussions/124300)
- [Discussion 124286](https://github.com/orgs/community/discussions/124286)
- [Discussion 124280](https://github.com/orgs/community/discussions/124280)
- [Discussion 124276](https://github.com/orgs/community/discussions/124276)
- [Discussion 124273](https://github.com/orgs/community/discussions/124273)
- [Discussion 124272](https://github.com/orgs/community/discussions/124272)
- [Discussion 124271](https://github.com/orgs/community/discussions/124271)
- [Discussion 124242](https://github.com/orgs/community/discussions/124242)
- [Discussion 124233](https://github.com/orgs/community/discussions/124233)
- [Discussion 124231](https://github.com/orgs/community/discussions/124231)
- [Discussion 124224](https://github.com/orgs/community/discussions/124224)
- [Discussion 124223](https://github.com/orgs/community/discussions/124223)
- [Discussion 124222](https://github.com/orgs/community/discussions/124222)
- [Discussion 124218](https://github.com/orgs/community/discussions/124218)
- [Discussion 124217](https://github.com/orgs/community/discussions/124217)
- [Discussion 124215](https://github.com/orgs/community/discussions/124215)
- [Discussion 124213](https://github.com/orgs/community/discussions/124213)
- [Discussion 124209](https://github.com/orgs/community/discussions/124209)
- [Discussion 124207](https://github.com/orgs/community/discussions/124207)
- [Discussion 124204](https://github.com/orgs/community/discussions/124204)

I believe addressing this could enhance the quality of our community discussions. Thank you!

**Comment by davevad93** [comment]
Hi @Tgentil ,

Yes, those spam posts are very annoying, every time I enter here in the discussions more than half of the posts are spam from airlines, streaming services or some kind of advertising. I've reported all the ones I have seen during the last few hours as spam to the Community administrators, I hope they can take measures to decrease the amount of spam because in my opinion it worsens the Community environment and the users experience.

**Comment by Akash1134** [comment, marked as answer]
👋🏻 Hi @Tgentil and @davevad93 ,

We really appreciate you flagging this and thank you for taking the time to compile the discussions 🙏 

The above threads have been actioned and we have reported the issue internally to prevent further spam in the community.

Thanks!

**Comment by rickhanlonii** [comment]
@Akash1134 is there anything more that can be done here? This spam is inundating the react repo.

**Reply by Akash1134** [reply]
Hi @rickhanlonii , Thanks for reporting! We have highlighted this to the concerned team internally and the spammers have been flagged to prevent this further. Location: [facebook/react](https://github.com/facebook/react/issues).
